duckduckgo/Android
# Bump Privacy Dashboard to 5.0.0

Task/Issue URL: https://app.asana.com/0/1206594217596623/1207921337238327/f

### Description

Bumps Privacy Dashboard version to 5.0.0 with changes to the gray state

Parent task: https://app.asana.com/0/72649045549333/1207737672183274/f

TL;DR: Introduces a new green state to the Privacy Dashboard for when nothing is blocked and only affiliated domains are allowlisted. Updates the copy of the existing gray state. **There are no changes on the native side.**

### Steps to test this PR

Repeat the steps for each of the sites in the table below:

1. Visit [URL]
2. Open Privacy Dashboard
3. Check that dashboard screen is similar to [Screenshot]
4. Check that [Things to note] are all true
5. Click dashboard nav links (e.g. "Connection is encrypted", "Third party requests loaded") to check that they're working

| URL | Screenshot | Things to note |
| - | - | - |
| https://ebay.de | <img width="387" alt="Screenshot 2024-08-01 at 15 37 45" src="https://github.com/user-attachments/assets/33fbf5ec-9f96-4a50-94ba-0edd34a4761e">| - Icon shows green checkmark<br> - "No Tracking Requests Found" nav link is NOT visible |
| https://amazon.co.uk |<img width="388" alt="Screenshot 2024-08-01 at 15 38 45" src="https://github.com/user-attachments/assets/e64c06d7-b04b-4380-892b-35e9c30473ff">| - Icon shows red exclamation icon<br> - "No Tracking Requests Found" nav link is NOT visible |
| https://reddit.com |<img width="388" alt="Screenshot 2024-08-01 at 15 35 45" src="https://github.com/user-attachments/assets/6f9a515f-8803-41d6-9836-710d9e9a0cf9">| - Icon shows grey info icon<br> - "No Tracking Requests Found" nav link is NOT visible |
| https://cnn.com |<img width="387" alt="Screenshot 2024-08-01 at 15 37 30" src="https://github.com/user-attachments/assets/1bf93213-50e6-4d1d-987e-020dc44dfdea">| - Multiple blocked icons shown<br> - "Requests Blocked from Loading" nav link IS visible |

### Notes
If any of the sites below do not show the expected screens, try refreshing the page, as trackers may be loaded alternately on reload. If the issue persists, please comment below with which sites had a mismatch. It doesn't mean the dashboard is broken, but it does mean I'll have to find a different site to test that screen. Different platforms may load different blockers, which leads to the dashboard not always showing the same status across all platforms and device configurations.
